### DISCUSSION TITLE
トラックボールが動作しなくなりました。

### DISCUSSION THREAD

**Discussion by fujitaka08** [Yowkees/keyball#173] [Q&A] [answered]
先日keyball44を購入させていただき、組み立て後数日は問題なく動作していたのですが、トラックボールが動作しないという状態になりました。
お手数ですが、解決策を教えていただけないでしょうか。
PCの再起動、ファームウェアの書き直し、基板を差しなおすということを実施したのですが解決しませんでした。
申し訳ありませんが、よろしくお願いいたします。

**Comment by fujitaka08** [comment, marked as answer]
自己解決しました。
pro microのはんだ不良が原因でした。
お騒がせしました。

**Comment by Yowkees** [comment]
ご連絡が遅くなり申し訳ありませんでした。
センサーICやジャンパの半田再確認をご提案予定でしたが、ProMicroでしたか！
解決できて良かったです。
今後ともどうぞよろしくお願いいたします。

**Reply by fujitaka08** [reply]
ご連絡ありがとうございます。
私も最初はそのあたりかなと思い、そこから見直したのですが、復旧しなかったためPro Microを再はんだしてみました。
キーは生きていたのであまり疑っていなかったのですが、不具合ポイントが見つかってよかったです。
